code/__DEFINES/food.dm

```diff
@@ -14,6 +14,7 @@
 #define BREAKFAST (1<<13)
 #define CLOTH (1<<14)
 #define NUTS (1<<15)
+#define ETHEREAL (1<<16) //i don't know what the fuck (1<<16) means but there is a pattern here so lets just go with that i guess
```

Comment:
This comment is unnecessary.
And to answer your comment, that's just how flags are defined, and you were correct in following the pattern.